app/main.py

```diff
@@ -1,3 +1,62 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: ["Distance", float]) -> "Distance":
```

Comment:
```suggestion
    def __iadd__(self, other: Union["Distance", float]) -> "Distance":
```
or
```suggestion
    def __iadd__(self, other: "Distance" | float) -> "Distance":
```
Fix in all places

Reply:
done